1.0/docs/devguide/properties.md

```diff
@@ -422,70 +422,42 @@ Example:
       </script>
     </dom-module>
 
-### Deep sub-property observation {#deep-observation}
-
-To call an observer when any (deep) sub-property of an
-object changes, specify a path with a wildcard (`*`).
-
-When you specify a path with a wildcard, the argument passed to your
-observer is a change record object with the following properties:
+### Observe array mutations
```

Comment:
Be sure to keep the original fragment identifier after the title, so existing links don't break:

    ### Observe array mutations {#array-observation}

It's good practice to always specify a fragment identifier for headings. Otherwise one gets auto-generated from the heading text, and will change if you ever reword the heading.)